src/Glpi/Form/DefaultFormsManager.php

```diff
@@ -0,0 +1,331 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Form;
+
+use AbstractRightsDropdown;
+use Glpi\Form\AccessControl\ControlType\AllowList;
+use Glpi\Form\AccessControl\ControlType\AllowListConfig;
+use Glpi\Form\AccessControl\FormAccessControl;
+use Glpi\Form\Destination\CommonITILField\ContentField;
+use Glpi\Form\Destination\CommonITILField\RequestTypeField;
+use Glpi\Form\Destination\CommonITILField\RequestTypeFieldConfig;
+use Glpi\Form\Destination\CommonITILField\RequestTypeFieldStrategy;
+use Glpi\Form\Destination\CommonITILField\SimpleValueConfig;
+use Glpi\Form\Destination\CommonITILField\TitleField;
+use Glpi\Form\Destination\FormDestination;
+use Glpi\Form\Destination\FormDestinationTicket;
+use Glpi\Form\QuestionType\QuestionTypeItemDropdown;
+use Glpi\Form\QuestionType\QuestionTypeLongText;
+use Glpi\Form\QuestionType\QuestionTypeObserver;
+use Glpi\Form\QuestionType\QuestionTypeShortText;
+use Glpi\Form\QuestionType\QuestionTypeUrgency;
+use Glpi\Form\Tag\AnswerTagProvider;
+use ITILCategory;
+use Location;
+use Ticket;
+
+final class DefaultFormsManager
+{
+    private AnswerTagProvider $answer_tag_provider;
+
+    public function __construct()
+    {
+        $this->answer_tag_provider = new AnswerTagProvider();
+    }
+
+    public function createDefaultForms(): void
+    {
+        if (countElementsInTable(Form::getTable()) > 0) {
+            return;
+        }
+
+        $this->createIncidentForm();
+        $this->createRequestForm();
+    }
+
+    private function createIncidentForm(): void
+    {
+        // Create form
+        $form = $this->createForm(
+            name: __('Incident'),
+            description: __("Declare an incident"),
+            icon: 'ti ti-exclamation-circle',
+        );
+
+        // Add questions
+        $this->setQuestions($form, [
+            $this->getUrgencyQuestionData(),
+            $this->getCategoryQuestionData(),
+            // TODO: associated items (not possible yet)
+            $this->getWatchersQuestionData(),
+            $this->getLocationQuestionData(),
+            $this->getTitleQuestionData(),
+            $this->getDescriptionQuestionData(),
+        ]);
+
+        // Find title and description question tags, needed to configure the created ticket
+        $title_tag = $this->answer_tag_provider->getTagForQuestion(
+            $form->getNthQuestion(4)
+        );
+        $description_tag = $this->answer_tag_provider->getTagForQuestion(
+            $form->getNthQuestion(5)
+        );
```

Comment:
This piece of code is not really resilient to default form data changes. It could probably be improved.